packages/web-app-files/src/views/spaces/DriveRedirect.vue

```diff
@@ -6,16 +6,11 @@
 
 <script lang="ts">
 import { computed, defineComponent, unref } from 'vue'
-import { useRoute, useRouter, useStore, useUserStore } from '@ownclouders/web-pkg'
+import { useRoute, useRouter, useStore } from '@ownclouders/web-pkg'
 import { AppLoadingSpinner } from '@ownclouders/web-pkg'
-import { urlJoin } from '@ownclouders/web-client/src/utils'
 import { createFileRouteOptions } from '@ownclouders/web-pkg'
 import { createLocationSpaces } from '@ownclouders/web-pkg'
 
-// 'personal/home' is used as personal drive alias from static contexts
-// (i.e. places where we can't load the actual personal space)
-const fakePersonalDriveAlias = 'personal/home'
```

Comment:
This is still needed for the nightwatch acceptance tests, see
- https://github.com/owncloud/web/blob/46ba0238543c6f8930932da6cc570bf84f38e8e8/packages/web-pkg/src/router/deprecated.ts#L56 and
- https://github.com/owncloud/web/blob/46ba0238543c6f8930932da6cc570bf84f38e8e8/packages/web-pkg/src/router/deprecated.ts#L66

As soon as we remove the nightwatch acceptance tests from this repo, those two routes and the fakePersonalDriveAlias handling here can be removed. `personal/home` has nothing to do with the CERN homeFolder feature.